app/views/groups/index.html.erb

```diff
@@ -0,0 +1,30 @@
+<%= render 'shared/navbar', group: @groups %>
+<button class="back"><%= link_to "back", groups_path  %></button>
+
+<div class="group_main_div">
+<h2 class="group_main_title">All Category</h2>
+
+<section class="main_group_section">
+<div class="groups_section">
+  <% @groups.each do |group| %>
+    <div>
+      <%= image_tag(group.icon || '', class: "group_image") %>
+    </div>
```

Comment:
- [ ] Dear @lawrahkonwea, Please keep in mind the concept of semantics and CEO while working with HTML codes, And as part of that for all the images we use, it would be great to add a `alt` text. This is great for screen readers and when we don't have an image in place.

### Your design when the category image is missing 
![image](https://github.com/lawrahkonwea/Budget_APP/assets/69077061/52f97166-6620-412d-9186-a260bc5a5510)